CHANGELOG.md

```diff
@@ -11,6 +11,10 @@
   * `ThreeDSecureV2LabelCustomization`
   * `ThreeDSecureV2TextBoxCustomization`
   * `ThreeDSecureV2ToolbarCustomization`
+* Add `PayPalCheckoutRequest`
+* Add `PayPalVaultRequest`
+* Add `tokenizePayPalAccount` method to `PayPalClient`
+* Add `requestBillingAgreement` to `PayPalCheckoutRequest`
```

Comment:
Was this option just not available before?

Reply:
Yeah we didn't have the option to checkout with vault on Android before - we were only ever setting a `billingAgreementDescription` during the `requestBillingAgreement()` (vault) flow. We added this flag to align with iOS.